packages/webapp/src/containers/Consent/locales/fr/Worker.Consent.md

```diff
@@ -1,4 +1,4 @@
-##### Version 4.0:
+##### Version 5.0:
 
 #### Formulaire de consentement éclairé et politique de confidentialité (version pour ouvrières et ouvriers)
```

Comment:
```suggestion
#### Formulaire de consentement éclairé et politique de confidentialité (version pour travailleurs et travailleuses agricoles)
```
Since these are the words we use for the roles in the application